date-time/src/commonTest/kotlin/com/splendo/kaluga/datetime/timer/Extensions.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.datetime.timer
+
+import kotlinx.coroutines.TimeoutCancellationException
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.withTimeout
+import kotlin.time.Duration
+
+suspend fun <T> Flow<T>.captureFor(duration: Duration): List<T> {
```

Comment:
This can live inside base module?

Reply:
actually I was half expecting a comment that there is a function in standard lib that does exactly the same..

by base module you mean `test-utils` or do you think there is a use case outside testing?

Reply:
Can be useful in future, who knows

Reply:
will move to `test-utils`

Reply:
done